openstack/CleanWlmDatabase/historical_data_cleanup/DeleteHistoricalRecords.py

```diff
@@ -0,0 +1,123 @@
+import os
+import time
+import configparser
+import re
+import mysql.connector
+
+# Get records to be deleted using select query
+# Write those records into file
+# Run delete on those records
+
+batchSize=1000
+
+# check if log directory is present, if not create one
+LOG_DIR="./LOGS"
+if not os.path.exists(LOG_DIR):
+   os.mkdir(LOG_DIR)
+
+# Create Logfile
+timestr = time.strftime("%Y%m%d-%H%M%S")
+LOG_FILE_NAME="deletehistoricalrecord-" + timestr + ".log"
+LOG_FILE_PATH=LOG_DIR + "/" + LOG_FILE_NAME
+logfile = open(LOG_FILE_PATH,"w")
+
+# DB connection 
+## Grab fields from the conf file
+config = configparser.ConfigParser()
+config.read('/etc/workloadmgr/workloadmgr.conf')
```

Comment:
let's provide the config file path as a parameter and use the provide file path as the wlm config file,
we can set `/etc/workloadmgr/workloadmgr.conf` as default path in case no cmd args provided